RoadTripMoustache/kana_to_kanji
# #100 Implement kana list and list tile

## 📖 Description


Introduce `KanaListTile` and also refactor `KanaList`.

### ⁉️ Related Issues




Closes #100 


### 🖼️ Screenshots:


<details>
    <summary>Demo</summary>     

https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/fa3051a7-bfbc-45fc-bba0-5823bc123ba2

</details>

<details><summary>Search demonstration</summary>

https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/bfeee28a-6612-4499-bd3f-b7f14d2af5d6

</details>
### 🧪 How to test the change?




1. Open the application
2. Go to the _Glossary_ tab
3. See changes you can also go to the _Katakana_ section

### ☑️ Checklist before requesting a review
